compiler/crates/relay-config/src/typegen_config.rs

```diff
@@ -58,6 +58,25 @@ pub struct CustomScalarTypeImport {
     pub path: PathBuf,
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone)]
+#[serde(untagged)]
+pub enum LiveResolverContextTypeInput {
+    Path(LiveResolverContextTypeInputPath),
+    Package(LiveResolverContextTypeInputPackage),
+}
+
+#[derive(Default, Debug, Serialize, Deserialize, Clone)]
+pub struct LiveResolverContextTypeInputPath {
+    pub name: String,
```

Comment:
Can we use `StringKey` here to avoid having to intern the string each time we use it?